src/report/mod.rs

```diff
@@ -130,12 +136,14 @@ pub fn gen<W: ReportWriter + Display>(ex_name: &str, dest: &W, config: &Config)
     )?;
     dest.write_string(
         "config.json",
-        serde_json::to_string(&ex.serializable())?.into(),
+        serde_json::to_string(&ex)?.into(),
         &mime::APPLICATION_JSON,
     )?;
+
+    let shas: HashMap<_, _> = shas.into_iter().map(|(r, s)| (r.url(), s)).collect();
     dest.write_string(
         "shas.json",
-        serde_json::to_string(&ex.shas.lock().unwrap().inner())?.into(),
+        serde_json::to_string(&shas)?.into(),
         &mime::APPLICATION_JSON,
     )?;
```

Comment:
I'm trying to puzzle out why we actually upload the shas.json. It was apparently (re-)added in https://github.com/rust-lang-nursery/crater/pull/92, but I can't see it actually being referenced anywhere in reports and I can't find any discussion of a shas.json on rust-lang/rust or this repo which would indicate people using it. What do you think about just not uploading it? We append the sha to the end of github repo names anyway.

We can always add it back and document why it exists if someone complains :)